### ISSUE TITLE
RcloneBrowser has problems accessing rclone.

### ISSUE THREAD

**Issue by HJSC00** [mmozeiko/RcloneBrowser#70]
Hi, 

The RcloneCrowser was working fine until yesterday, but I've been trying to mount the drive of Google Drive for Plex to have access to my multimedia content. Well, the assembly worked and everything else, however, now the RcloneBrowser does not open, actually it opens and closes automatically afterwards.

I'll leave the steps I followed:

**01. go into root mode:**
> sudo -s

**02. I create the mount script:**
> nano /lib/systemd/system/google.service

**03. Then paste (right click):**
```
[Unit]
Description=Mount GSuite to /mnt/GSuite
After=syslog.target local-fs.target network.target
[Service]
Type=simple
User=root
ExecStartPre=-/bin/mkdir /mnt/GSuite
ExecStart=/usr/bin/rclone mount \
  --config /home/hjsc/.config/rclone/rclone.conf \
  --read-only \
  --allow-other \
  --allow-non-empty \
  --buffer-size 64M \
  GSuite: /mnt/GSuite
ExecStop=/bin/fusermount -u //mnt/GSuite
ExecStop=/bin/rmdir /mnt/GSuite
Restart=always
[Install]
WantedBy=multi-user.target
```
**04. I enabled the service:**
> systemctl enable google.service

**05. Reload the service and reboot the system:**
> systemctl daemon-reload
> exit
> sudo reboot
> 
I had never manually or specified the rclone.conf file before. From the beginning I installed rclone, then RcloneBrowser and for it I made all the configuration of my Google Drive account.

Thanks in advance, friends!

**Comment by mmozeiko** [comment]
Are you saying that when service is running then RcloneBrowser crashes at startup?
So if you try stopping service (`systemctl stop google.service`) then RcloneBrowser will start just fine?

Btw I suggest you to put your custom .service files in /etc/systemd/system folder, and not in system /lib... folder which can be removed/overwritten when you install or remove some other package.

**Comment by HJSC00** [comment]
No, even stopping the service, RcloneBrowser keeps closing automatically. However, when running RB as sudo (sudo rclone-browser) it opens normally. What will happen to leads to this ?! I unmounted the drive and uninstalled the plex, because I could not configure it to not transcode and my vps is very weak.

Do you have any idea how to get back to normal? Apparently, RcloneBrowser only run if it is by sudo.

**Comment by mmozeiko** [comment]
If you have run RcloneBrowser with sudo then it might be that it created its config file with root permissions. And now it cannot read or write it with your user permissions. Try to delete `~/.config/rclone-browser` folder and re-run it without sudo.

**Comment by HJSC00** [comment]
That was exactly it. The access to rclone.conf file was only for root. All right now. =)
